ReportingServicesTools.psd1

```diff
@@ -7,7 +7,7 @@
 	RootModule = 'ReportingServicesTools.psm1'
 	
 	# Version number of this module.
-	ModuleVersion = '0.0.0.8'
+	ModuleVersion = '0.1.0.9'
```

Comment:
Ah thanks, not quite used to working on Git Projects with other people yet. Will remember that and not touch it.